src/cmap/connection_pool.ts

```diff
@@ -28,7 +28,6 @@ import { CancellationToken, TypedEventEmitter } from '../mongo_types';
 import type { Server } from '../sdam/server';
 import {
   type Callback,
-  eachAsync,
```

Comment:
This utils function is now also used nowhere in the codebase, do we want to remove it? It uses callbacks, and IIUC we no longer want to add any new callback code to the driver.

Reply:
Good point. I agree that removing it is likely the way we want to go here. @nbbeeken @baileympearson thoughts?

Reply:
Go for it